core/src/test/java/de/bwaldvogel/mongo/backend/aggregation/ExpressionTest.java

```diff
@@ -1265,7 +1274,7 @@ void testEvaluateStrLenCP() throws Exception {
         assertThat(Expression.evaluate(json("$strLenCP: '$a'"), json("a: 'value'"))).isEqualTo(5);
         assertThat(Expression.evaluate(json("$strLenCP: 'cafétéria'"), json(""))).isEqualTo(9);
         assertThat(Expression.evaluate(json("$strLenCP: '$a'"), json("a: '$€λA'"))).isEqualTo(4);
-        assertThat(Expression.evaluate(json("$strLenCP: '寿司'"), json(""))).isEqualTo(2);
+        assertThat(Expression.evaluate(json("$strLenCP: '\u5BFF\u53F8'"), json(""))).isEqualTo(2);
```

Comment:
Windows is a bit funny about these characters. I've replaced them with their unicode equivalents.

Reply:
if it solves issues on Window, lets keep it as you suggested